### PR TITLE
Add arrayFilter option

### PR DESCRIPTION
**Pull request by datacas**
Add option arrayFilters to allow to use the arrayfilters of version 3.6 of Mongo.

https://docs.mongodb.com/manual/reference/command/update/#update-command-arrayfilters
https://docs.mongodb.com/php-library/current/reference/method/MongoDBCollection-updateMany/

Hello!

* Type: bug fix | new feature | code quality | documentation
* Link to issue:

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/incubator/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change:

Thanks

### CODE REVIEW

File: Library/Phalcon/Db/Adapter/MongoDB/Operation/Update.php [hunk 1]
```diff
@@ -136,6 +136,10 @@ public function execute(Server $server)
             'upsert'=>$this->options['upsert'],
         ];
 
+        if (isset($this->options['arrayFilters'])) {
+            $updateOptions['arrayFilters'] = $this->options['arrayFilters'];
+        }
+
         $bulkOptions=[];
 
         if (isset($this->options['bypassDocumentValidation'])&&Functions::serverSupportsFeature(
```
Comment (stamster) [inline Library/Phalcon/Db/Adapter/MongoDB/Operation/Update.php]: spaces around `&&` would be much nicer here.

File: Library/Phalcon/Db/Adapter/MongoDB/Operation/Update.php [hunk 2]
```diff
@@ -136,9 +136,13 @@ public function execute(Server $server)
             'upsert'=>$this->options['upsert'],
```
Comment (Jeckerson) [inline Library/Phalcon/Db/Adapter/MongoDB/Operation/Update.php]: Make it PSR-2 compatible (https://www.php-fig.org/psr/psr-2/)

File: Library/Phalcon/Db/Adapter/MongoDB/Operation/Update.php [hunk 3]
```diff
@@ -136,9 +136,13 @@ public function execute(Server $server)
...
         ];
 
+        if (isset($this->options['arrayFilters'])) {
+            $updateOptions['arrayFilters'] = $this->options['arrayFilters'];
+        }
+
         $bulkOptions=[];
```
Comment (Jeckerson) [inline Library/Phalcon/Db/Adapter/MongoDB/Operation/Update.php]: Make it PSR-2 compatible (https://www.php-fig.org/psr/psr-2/)

**Comment by datacas** [comment]
Is any change necessary?

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**